docs/users-guide/expressions.md

```diff
@@ -0,0 +1,102 @@
+## Writing expressions in the notebook editor
+
+Custom expressions are a way to create more advanced filters and aggregations, or to add custom columns to your custom question. These expressions are accessible in the notebook editor of custom questions when clicking the button to add a new filter, a new metric in the Summarize area, or when creating a new custom column.
+
+### How to write expressions
+
+In each of these three places, you can:
+
+- Use parentheses to group parts of your expression.
+- Use basic mathematical operators: `+`, `-`, `*` (multiply), `/` (divide).
```

Comment:
I think it would be good to add some commentary one what types of fields you can use mathematical operators with. e.g. Can I subtract a bigint field from a string field? What happens when I do? Can I do math on timestamp fields? (Doesn't seem like with current `HEAD`.)